docs/Deploying/fleetctl-agent-updates.md

```diff
@@ -1,8 +1,10 @@
 # Self-managed agent updates 
 
-Fleet's [Orbit osquery updater](https://github.com/fleetdm/fleet/tree/main/orbit) by default utilizes the public Fleet update repository.
+[Orbit](https://github.com/fleetdm/fleet/tree/main/orbit) will periodically check the public Fleet update repository update itself, Fleet Desktop, and/or osquery 
```

Comment:
I think there is a typo here. I believe it should read **Orbit will periodically check the public Fleet update repository to update itself, Fleet Desktop, and/or osquery